tests/sample/map.c

```diff
@@ -183,7 +157,51 @@ SEC("xdp_prog") int test_maps(struct xdp_md* ctx)
     TEST_MAP(LRU_HASH, LRU);
     TEST_MAP(LRU_PERCPU_HASH, LRU);
 
-    TEST_MAP(QUEUE, PUSH_POP);
-    TEST_MAP(STACK, PUSH_POP);
+    // TEST_MAP(QUEUE, PUSH_POP) is too large for clang to inline, so inline manually.
```

Comment:
I checked that Cilium code use `__always_inline` and cilium code has many inline functions. Maybe we can try this and see if clang inlines the code?

Also, I think the function was already getting inlined when `__attribute__((always_inline))` was used?

Reply:
reverted